crates/subspace-networking/examples/relayed-requests.rs

```diff
@@ -48,20 +58,9 @@ async fn main() {
 
     let config_2 = Config {
         allow_non_globals_in_dht: true,
-        request_response_protocols: vec![PiecesByRangeRequestHandler::create(|req| {
```

Comment:
Why do we need a mock request here instead of the pieces-by-range request?

Reply:
Answered above